ntbs-service/wwwroot/source/Components/DateComparison.ts

```diff
@@ -9,17 +9,26 @@ const DateComparison = Vue.extend({
             dateWarnings: []
         }
     },
+    mounted: function () {
+        // wait for children to mount
+        this.$nextTick(function ()
+        {
+            for (let i = this.dates.length; i >= 0; i--) {

```

Comment:
Cycling backwards over the dates gives the date warnings in the most intuitive direction.